src/net_processing.cpp

```diff
@@ -1162,7 +1162,7 @@ void static ProcessGetData(CNode* pfrom, const Consensus::Params& consensusParam
     }
 }
 
-uint32_t GetFetchFlags(CNode* pfrom, const CBlockIndex* pprev, const Consensus::Params& chainparams) {
+uint32_t GetFetchFlags(CNode* pfrom) {
```

Comment:
Any idea *why* these were being passed?
Looked back in history and to me it seems that `CBlockIndex* pprev, const Consensus::Params& chainparams` were never used at all, since introduced in b8a97498df1e83f8dcc49bc3fa4344f9e9799242.
@sipa @NicolasDorier  maybe?